src/character.h

```diff
@@ -484,6 +484,7 @@ class Character : public Creature, public visitable<Character>
         std::vector<bionic> my_bionics;
 
     protected:
+        virtual void on_stat_change( const std::string &, int ) override {};
```

Comment:
Agreed. Passing strings around, where enums are sufficient is considered a code smell, but string-based approach seems to be widespread in the code, so I used it. I'll gladly replace them with enums.